metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery.py

```diff
@@ -722,20 +751,40 @@ def _process_table(
             self.report.report_dropped(table_identifier.raw_table_name())
             return
 
-        table.columns = self.get_columns_for_table(
-            conn, table_identifier, self.config.column_limit
-        )
-        if not table.columns:
+        table.column_count = len(columns)
```

Comment:
Minor: Any chance that columns is None??? This would throw

Reply:
(i see above it is set to [] if not exists - nice!)

Reply:
I think mypy should complain if this would be possible